consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/client/mock/MockJudgementWarrantService.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.moj.sdt.cmc.consumers.client.mock;
+
+import org.springframework.stereotype.Service;
+import uk.gov.moj.sdt.cmc.consumers.api.IJudgementWarrantService;
+import uk.gov.moj.sdt.cmc.consumers.request.JudgementWarrantRequest;
+import uk.gov.moj.sdt.cmc.consumers.response.JudgementWarrantResponse;
+
+@Service("MockJudgementWarrantService")
+public class MockJudgementWarrantService implements IJudgementWarrantService {
+
+    @Override
+    public JudgementWarrantResponse judgementWarrantRequest(String authorization,
+                                                            String serviceAuthorization,
+                                                            String idamId,
+                                                            String sdtRequestId,
+                                                            JudgementWarrantRequest judgementWarrantRequest) {
+        JudgementWarrantResponse response = new JudgementWarrantResponse();
+        response.setWarrantNumber("123456");
+        response.setFee(45678L);
```

Comment:
Might be good to set at least one of the judgment related fields as well, just to distinguish it from a warrant.

Reply:
done